src/base/memory_monitor_sr.hpp

```diff
@@ -40,12 +47,18 @@ namespace cubmem
       memory_monitor &operator = (const memory_monitor &) = delete;
       memory_monitor &operator = (memory_monitor &&) = delete;
 
+    public:
+      size_t get_allocated_size (const char *ptr);
```

Comment:
How is this used? I can't find stuff on the Jira issue or PR. To review the correctness. Does it just return the size of "user space" in the allocated space? When is it used?

Reply:
Please remind me if it's described in the meeting. Better to describe it in the Jira issue. Please give me a slack. For reviewers, it's hard to remember every single thing.

Reply:
Ah.. probably it's used to subtract the memory usage, that occurred to me.

Reply:
I write the purpose of this function since the first time of issue opening..
"This function is used to getting user-requested size of memory for wrapping function of realloc."

Reply:
I wondered what happens if it returns 0, so I wanted to know when it is used, but I couldn't find stuff about this. Let's see the a following PR.